pom.xml

```diff
@@ -154,7 +154,144 @@
+					<excludes>
+						<exclude>**/IT*.java</exclude>
+					</excludes>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-failsafe-plugin</artifactId>
+				<version>3.0.0-M7</version>
+				<executions>
+					<!--
+              Invokes both the integration-test and the verify goals of the
+              Failsafe Maven plugin
+          -->
+					<execution>
+						<id>integration-tests</id>
+						<goals>
+							<goal>integration-test</goal>
+							<goal>verify</goal>
+						</goals>
+						<configuration>
+							<!--
+                  Skips integration tests if the value of skip.integration.tests
+                  property is true
+              -->
+							<skipTests>${skip.integration.tests}</skipTests>
+						</configuration>
+					</execution>
+				</executions>
+			</plugin>
 		</plugins>
+
+		<resources>
+			<!--
+          Placeholders that are found from the files located in the configured resource
+          directories are replaced with the property values found from the profile
+          specific configuration file.
+      -->
+			<resource>
+				<filtering>true</filtering>
+				<directory>src/main/resources</directory>
+			</resource>
+		</resources>
 	</build>
 
+	<profiles>
+		<!-- The Configuration of the development profile -->
+		<profile>
+			<id>dev</id>
+			<activation>
+				<activeByDefault>true</activeByDefault>
+			</activation>
+			<properties>
+				<!--
+            Specifies the build.profile.id property that must be equal than the name of
+            the directory that contains the profile specific configuration file.
+            Because the name of the directory that contains the configuration file of the
+            development profile is dev, we must set the value of the build.profile.id
+            property to dev.
+        -->
+				<build.profile.id>dev</build.profile.id>
+				<!--
+            Only unit tests are run when the development profile is active
+        -->
+				<skip.integration.tests>true</skip.integration.tests>
+				<skip.unit.tests>false</skip.unit.tests>
+			</properties>
+		</profile>
+		<!-- The Configuration of the integration-test profile -->
+		<profile>
+			<id>integration-test</id>
+			<properties>
+				<!--
+            Specifies the build.profile.id property that must be equal than the name of
+            the directory that contains the profile specific configuration file.
+            Because the name of the directory that contains the configuration file of the
+            integration-test profile is integration-test, we must set the value of the
+            build.profile.id property to integration-test.
+        -->
+				<build.profile.id>integration-test</build.profile.id>
+				<!--
+            Only integration tests are run when the integration-test profile is active
+        -->
+				<skip.integration.tests>false</skip.integration.tests>
+				<skip.unit.tests>true</skip.unit.tests>
```

Comment:
Is there a reason why we skip unit tests under the integration test profile? 

If this is the desired behavior, maybe we can run `./mvnw clean verify` with both profiles once to avoid the error we're currently getting with the CI flow